pkg/processes.go

```diff
@@ -0,0 +1,66 @@
+package batchinsights
+
+import (
+	"github.com/shirou/gopsutil/process"
+)
+
+// love to program in go
+func contains(xs []string, str string) bool {
+
+	for _, x := range xs {
+		if str == x {
+			return true
+		}
+	}
+
+	return false
+
+}
+
+func ListProcesses(processNames []string) ([]*ProcessPerfInfo, error) {
+	pids, err := process.Pids()
+	if (err != nil) {
+		return nil, err
+	}
+
+	ps := []*ProcessPerfInfo {}
+	for _, pid := range pids {
+
+		// if err != nil, process has probably disappeared, continue on
+		if p, err := process.NewProcess(pid); err == nil {
+
+			name, err := p.Name()
+			if err != nil {
+				// process might have disappeared
+				continue
+			}
+
+			// check if we should include it
+			if !contains(processNames, name) {
+				continue
+			}
+
+			cpuPercent, err := p.CPUPercent()
```

Comment:
Does this work fine on windows? The main CPU usage has been implemented wrongly on windows(Or at least was when I made it, might have been fixed) I had to call wmi directly

Reply:
That's annoying...

Reply:
I saw a PR on the repo that might have fixed it recently so it might just be working fine